types/setting.go

```diff
@@ -359,6 +362,21 @@ var (
 			string(VolumeAttachmentRecoveryPolicyImmediate),
 		},
 	}
+	SettingDefinitionStatefulSetPodDeletePolicy = SettingDefinition{
```

Comment:
`StatefulSetNodeDownRecoveryPolicy`

Or we can simply do

`DeletePodInStatefulSetWhenNodeDown`. Do we need a `choice` here? Or a boolean is enough.